Makefile

```
@@ -386,7 +386,19 @@ targets-group-2: $(GROUP_2_TARGETS)
 ## targets-group-3   : build some targets
 targets-group-3: $(GROUP_3_TARGETS)
 
-## targets-group-rest: build the rest of the targets (not listed in group 1, 2 or 3)
+## targets-group-4   : build some targets
+targets-group-4: $(GROUP_4_TARGETS)
```

checkmake (0.3.2)

[warning] Target "targets-group-4" should be declared PHONY.

(phonydeclared)